test/mongodb.test.js

```diff
@@ -116,6 +117,31 @@ describe('mongodb connector', function() {
         done();
       });
     });
+
+    it('should skip connect phase (lazyConnect = true)', function(done) {
+      var ds = getDataSource({
+        host: 'localhost',
+        port: 4,
+        lazyConnect: true
+      });
+      initialize(ds, function(err) {
+        should.not.exist(err);
+        done();
+      })
+    });
+
+    it('should report connection error (lazyConnect = false)', function(done) {
+      var ds = getDataSource({
+        host: 'localhost',
```

Comment:
It's better to use `127.0.0.1` instead of localhost, to ensure we use IPv4 on all machines. Same applies to the other test above.